### PR TITLE
MOTR-111 MOTR-113 Changed team page and homepage description

### PR DESCRIPTION
**Pull request by ysyskim**
Key changes: 
1. changed the text from 'how activity' to 'how physical activity' on the homepage description
2. changed the text from 'perform preliminary characterization of' to 'characterize'
3. added Young's profile to the team page & Made it consistent with people's titles (i.e. removing 'Professor' and adding 'Ph.D.', 'M.S.')

### CODE REVIEW

File: src/lib/teamInfo.js [hunk 1]
```diff
@@ -94,26 +103,39 @@ const teamInfo = {
...
+      name: 'Jonathan N. Myers, Ph.D.',
+      title: 'Advisor',
+      image: JonImg,
+    },  
   ],
+  Alumni: [
+    {
+      name: 'Samir Akre',
+      title: 'Alumni',
+      //title: 'Alumni Software & Pipeline Developer',
```
Comment (jimmyzhen) [inline src/lib/teamInfo.js]: @ysyskim We generally don't retain unused code in comments. `git` gives us the ability to roll back changes if needed.

Comment (ysyskim) [reply src/lib/teamInfo.js]: This comment has been removed.

Comment (jimmyzhen) [reply src/lib/teamInfo.js]: Thanks!

**Comment by ysyskim** [comment]
> @ysyskim, please see review comment. Also, is the `src/assets/teamMemberPhotos/Myers.jpg` image file being used at all?

Yes I've removed the Myers.jpg file.

**Comment by kilodalton** [comment]
Merged MOTR-111 MOTR-113

**Pull request merged**